core/src/main/scala/chisel3/RawModule.scala

```diff
@@ -66,15 +66,15 @@ abstract class RawModule(implicit moduleCompileOptions: CompileOptions) extends
           if (id.isSynthesizable) {
             id.topBinding match {
               case OpBinding(_, _) =>
-                id.forceName(Some(""), default = "T", _namespace)
+                id.forceName(None, default = "_T", _namespace)
```

Comment:
I realized that my changes here make the "defaultPrefix" argument of `_computeName` redundant so I need to further clean that up.